kanidm_book/src/accounts_and_groups.md

```diff
@@ -141,6 +141,16 @@ Whether an account is currently a "person" or not can be identified from the "ac
 The presence of a "class: person" stanza indicates that this account may have
 "people" attributes.
 
+### Allowing people accounts to change their mail
+
+By default, Kanidm only allows a "person" account to change their legal name,
```

Comment:
Kanidm allow a person to change their legal name, display name, username .... :)

Reply:
I was just thinking of person-type attributes, which I thought were just mail and legalname, while display name, username, etc are account attributes (iiuc?)

You're right that the wording's bad. I updated it to not mention legal name specifically, and just call out that mail isn't in the default write set.